test/core/vm/actor/init_actor_test.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/init_actor.hpp"
+
+#include <gtest/gtest.h>
+#include "primitives/address/address_codec.hpp"
+#include "storage/hamt/hamt.hpp"
+#include "storage/ipfs/impl/in_memory_datastore.hpp"
+#include "testutil/cbor.hpp"
+
+using fc::vm::actor::InitActorState;
+
+/** Init actor state CBOR */
```

Comment:
according to our practices, tests should be supplied with /** @given ... @when ... @then ... */ description.

Reply:
Lotus-compatible CBOR encoding tests are common and simple.
Big description would be excessive.
Most of them would just use `expectEncodeAndReencode`.